src/keycloak/admin/realmroles.py

```diff
@@ -0,0 +1,78 @@
+import json
+from collections import OrderedDict
+
+from keycloak.admin import KeycloakAdminBase, KeycloakAdminEntity
+
+ROLE_KWARGS = [
+    'description',
+    'id',
+    'realm_role',
+    'composite',
+    'composites',
+    'container_id',
+    'scope_param_required'
+]
+
+__all__ = ('to_camel_case', 'RealmRole', 'RealmRoles',)
```

Comment:
Is the to_camel_case method needed in __all__?

Reply:
Yes it is used in create function

Reply:
but `__all__` is all the things can be improted from this module, so I do not think camel case method is neede there